README.md

```diff
@@ -1,42 +1,32 @@
 # ONSAS.jl
 
----
 ## *An Open Nonlinear Structural Analysis Solver*
 
-| **Documentation** |**Status** |**License** |  **Community** |
-|:-----------------:|:----------------------------:|:-----------:| :-----------:|
-| [![docs-dev][dev-img]][dev-url] | [![CI][ci-img]][ci-url] [![codecov][cov-img]][cov-url] |  [![license][lic-img]][lic-url] | [![community][com-img]][com-url] |
-
+[![docs-dev][dev-img]][dev-url] [![CI][ci-img]][ci-url] [![codecov][cov-img]][cov-url] [![license][lic-img]][lic-url]
 
 [dev-img]: https://img.shields.io/badge/docs-stable-blue.svg
 [dev-url]: https://ONSAS.github.io/ONSAS.jl/dev/
 [ci-img]: https://github.com/ONSAS/ONSAS.jl/actions/workflows/CI.yml/badge.svg?branch=main
 [ci-url]: https://github.com/ONSAS/ONSAS.jl/actions/workflows/CI.yml?query=branch%3Amain
 [cov-img]: https://codecov.io/gh/ONSAS/ONSAS.jl/branch/main/graph/badge.svg
 [cov-url]: https://codecov.io/gh/ONSAS/ONSAS.jl
-[doi-img]: https://zenodo.org/badge/170120732.svg
-[doi-url]: https://zenodo.org/badge/latestdoi/
 [lic-img]: https://img.shields.io/github/license/mashape/apistatus.svg?maxAge=2592000
-[lic-url]: https://github.com/mvanzulli/ONSAS.jl/blob/main/LICENSE
-[com-url]: https://app.slack.com/client/T04QWNG5T2Q/C04R6TMDV0R
-[com-img]: https://user-images.githubusercontent.com/50339940/224228295-4ff7c201-975f-4e87-af51-86036a10f289.svg
+[lic-url]: https://github.com/ONSAS/ONSAS.jl/blob/main/LICENSE
 
 
+`ONSAS.jl` is a Julia implementation of the [ONSAS.m](https://github.com/ONSAS/ONSAS.m/) Structural Analysis Solver.
 
----
-## 💾 [Installing](https://julialang.github.io/Pkg.jl/v1/managing-packages/)
+## Installation
 
-`ONSAS.jl` is a registered Julia package and [as such you can install it](https://julialang.github.io/Pkg.jl/v1/managing-packages/) by activating the `pkg` mode (type `]`, and to leave it, type `<backspace>`),
+`ONSAS.jl` is a registered Julia package and [as such you can install it](https://julialang.github.io/Pkg.jl/v1/managing-packages/), from your Julia terminal, by activating the `pkg` mode (type `]`, and to leave it, type `<backspace>`),
```

Comment:
If this is intended for brand newcomers to Julia, maybe you can rely on the fact that nowadays you don't need to know about `pkg` mode at all?

just do `using ONSAS` then press enter to install it. next time `using` it won't ask the person to install it again.

```julia
julia> using Example
 │ Package Example not found, but a package named Example is available from a
 │ registry. 
 │ Install package?
 │   (@v1.9) pkg> add Example 
 └ (y/n/o) [y]: 
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
   Installed Example ─ v0.5.3
    Updating `~/.julia/environments/v1.9/Project.toml`
  [7876af07] + Example v0.5.3
    Updating `~/.julia/environments/v1.9/Manifest.toml`
  [7876af07] + Example v0.5.3
Precompiling project...
  1 dependency successfully precompiled in 1 seconds. 222 already precompiled.
```

Reply:
it's up to you really; i know that one target audience for this package are students who weren't exposed to Julia before, hence the comment.

Reply:
gracias! hecho en [75ffbc6](https://github.com/ONSAS/ONSAS.jl/pull/444/commits/75ffbc6177c602c86ae53ea80d4a5c6e4c2994f9)